chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -133,35 +139,55 @@ Zotero_Preferences.Export = {
 	},
 	
 	
-	updateQuickCopyHTMLCheckbox: function () {
+	updateQuickCopyUI: function () {
 		var format = document.getElementById('zotero-quickCopy-menu').value;
+		
 		var mode, contentType;
 		
-		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		[mode, format] = format.split('=');
 		[mode, contentType] = mode.split('/');
 		
+		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		checkbox.checked = contentType == 'html';
 		checkbox.disabled = mode != 'bibliography';
+		
+		var menulist = document.getElementById('locale-menu');
+		if (mode != 'bibliography') {
+			// maintain menu selection but show blank label
+			menulist.insertItemAt(0, '', menulist.value);
+			menulist.selectedIndex = 0;
+			menulist.disabled = true;
+		} else if (menulist.disabled == true) {
+			// restore original menu selection
+			let oldValue = menulist.selectedItem.value;
+			menulist.removeItemAt(0);
+			menulist.value = oldValue;
+			menulist.disabled = false;
+		}
```

Comment:
Should work now. Implemented in https://github.com/rmzelle/zotero/commit/8356602dba030ef9b1569b6278e9f6efc49b9874 and https://github.com/rmzelle/zotero/commit/d6da32888caa0576878e05360817e6e112db5867